packages/gatsby-plugin-image/src/node-apis/image-processing.ts

```diff
@@ -130,37 +116,24 @@ export async function writeImages({
 
 export async function writeImage(
   file: Node,
-  pathPrefix: string,
   args: SharpProps,
   reporter: Reporter,
   cache: GatsbyCache,
-  isFixed: boolean,
   filename: string
 ): Promise<void> {
   try {
-    if (args.tracedSVG) {
-      // These are mutually-exclusive options
-      args.base64 = false
-    }
     const options = { file, args, reporter, cache }
 
+    if (!PluginSharp.gatsbyImageProps) {
+      reporter.warn(`Please upgrade gatsby-plugin-sharp`)
```

Comment:
Should this be a panic instead?

Reply:
I don't think we need to panic. We panic too much. I think returning null or void is fine.